packages/ui-tree-browser/src/TreeBrowser/TreeCollection/index.js

```diff
@@ -76,7 +76,7 @@ class TreeCollection extends Component {
     /**
      * children of type TreeNode
      */
-    renderBeforeCollections: Children.oneOf([TreeNode]),
+    renderBeforeItems: Children.oneOf([TreeNode]),
```

Comment:
This change is because the "render stuff after the list" is called `renderAfterItems`